CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+##2016-07-XX (TBD) - Release 2.2.0
+### Summary
```

Comment:
MD formatting: for best renderer support, a header should be followed by a blank line.